node/rpc/src/chain_api.rs

```diff
@@ -69,10 +69,10 @@ where
     KS: KeyStore + Send + Sync + 'static,
     B: Beacon + Send + Sync + 'static,
 {
-    let (data_subscribe, _) = data.state_manager.chain_store().subscribe().await;
+    let (data_subscribe, ts) = data.state_manager.chain_store().subscribe().await;
     let index = chain::sub_head_changes(
         data_subscribe,
-        &data.state_manager.chain_store().heaviest_tipset().await,
+        &Some(ts),
```

Comment:
why is this an option if this is the only place it's used? Should subscribe just return the option then?